spec/policies/activity_policy_spec.rb

```diff
@@ -176,6 +185,11 @@
           it { is_expected.to forbid_action(:create_child) }
           it { is_expected.to forbid_action(:create_transfer) }
           it { is_expected.to forbid_action(:create_refund) }
+
+          it "permits an adjustment where the report is approved" do
+            expect(Report.for_activity(activity).any? { |r| r.state == "approved" })
+            is_expected.to permit_action(:create_adjustment)
```

Comment:
yes we could do that -- and probably it will be clearer. I saw that we already had a spec which happens to put the report in the necessary 'approved' state and rather than build more objects I tried to:

- use what I had to hand and avoid more setup
- make it clear on line 190 that the Report is in the approved state

but I think it's fair to say that the result is actually more confusing!

I will take another run at this...

Reply:
I've fixed this up. Turns out it was super-easy to add:

```
before do
  report.update(state: :approved)
end
```

thanks for bringing it up.